### ISSUE TITLE
Date Slider with two dates and floor is broken

### ISSUE THREAD

**Issue by Head** [angular-slider/angularjs-slider#610]
### Steps to reproduce
Create a slider with two dates and set options.floor

Demo: http://jsfiddle.net/kdysqp1v/2/

**Comment by Head** [comment]
Not using date objects but date.getTime() for all options works. But it's still a bug if communicated that it should work with dates.

**Comment by ValentinH** [comment]
Good catch. It needs to be address. I'll try to look into this when I have a bit of time...

**Comment by mg1075** [comment]
related (?): https://github.com/angular-slider/angularjs-slider/issues/614
